web.py

```diff
@@ -14,3 +16,26 @@ def home():
 def sim(index):
     graph_json = SIMULATION[index]
     return graph_json
+
+
+app.config["UPLOADS"] = "/workspaces/MARLon/uploads"
+
+@app.route('/upload-file', methods=['POST'])
+def upload_file():
+    
+    if request.files['attackerFile']:
+        afile = request.files['attackerFile']
+    else:
+        print("Missing Attacker File")
+        return render_template('index.html', max_sim=len(SIMULATION) - 1)
+    if request.files['defenderFile']:
+        dfile = request.files['defenderFile']
+    else:
+        print("Missing Defender File")
+        return render_template('index.html', max_sim=len(SIMULATION) - 1)
+
+    print("Both files loaded")
+    afile.save(os.path.join(app.config["UPLOADS"], afile.filename))
+    dfile.save(os.path.join(app.config["UPLOADS"], dfile.filename))
```

Comment:
I think when the simulation gets wired up we won't even need to save this to the disk, we can deserialize the data directly. We'll figure it out when we get there so this is fine for now.